tests/apis/platformvm/api.test.ts

```diff
@@ -2561,4 +2585,528 @@ describe("PlatformVMAPI", (): void => {
     expect(mockAxios.request).toHaveBeenCalledTimes(1)
     expect(response).toBe(payload["result"])
   })
+
+  test("getRegisteredShortIDLink", async (): Promise<void> => {
+    const result: Promise<string> = api.getRegisteredShortIDLink(nodeID)
+    const payload: object = {
+      result: { address: addrA }
+    }
+    const responseObj: HttpResponse = {
+      data: payload
+    }
+
+    mockAxios.mockResponse(responseObj)
+    const response: string = await result
+
+    expect(mockAxios.request).toHaveBeenCalledTimes(1)
+    expect(response).toBe(addrA)
+  })
+
+  test("getAllDepositOffers", async (): Promise<void> => {
+    const result: Promise<DepositOffer[]> = api.getAllDepositOffers()
+    const payload: object = {
+      result: {
+        depositOffers: [
+          {
+            id: "1",
+            interestRateNominator: "2",
+            start: "3",
+            end: "4",
+            minAmount: "5",
+            minDuration: 6,
+            maxDuration: 7,
+            unlockPeriodDuration: 8,
+            noRewardsPeriodDuration: 9,
+            memo: "memo",
+            flags: 0
+          },
+          {
+            id: "2",
+            interestRateNominator: "2",
+            start: "3",
+            end: "4",
+            minAmount: "5",
+            minDuration: 6,
+            maxDuration: 7,
+            unlockPeriodDuration: 8,
+            noRewardsPeriodDuration: 9,
+            memo: "memo",
+            flags: 1
+          }
+        ]
+      }
+    }
+    const responseObj: HttpResponse = {
+      data: payload
+    }
+
+    mockAxios.mockResponse(responseObj)
+    const response: DepositOffer[] = await result
+
+    expect(mockAxios.request).toHaveBeenCalledTimes(1)
+    expect(response.length).toBe(2)
+    expect(response[0].id).toBe("1")
```

Comment:
Wouldn't it be better to add a loop to check all fields instead of hard coded checks? Something like:
```suggestion
    for (let i = 0; i < response.length; i++) {
      expect(response[i].id).toBe(payload["result"].depositOffers[i].id)
      expect(response[i].interestRateNominator).toStrictEqual(
        new BN(payload["result"].depositOffers[i].interestRateNominator)
      )
      expect(response[i].start).toStrictEqual(
        new BN(payload["result"].depositOffers[i].start)
      )
     ...
```

Same goes with the rest of the functions. Otherwise, great work. LGTM

Reply:
i thought it's not necessary since the only field changing is the the id and the flag.